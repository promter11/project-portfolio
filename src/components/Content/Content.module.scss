.main {
  width: 100%;
  padding-left: 32rem;
  font-family: "Public Sans", sans-serif;
}
