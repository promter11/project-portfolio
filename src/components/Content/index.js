import React from "react";

import Routes from "../../routes";

import styles from "./Content.module.scss";

const Content = ({ language }) => {
  return (
    <main className={styles.main}>
      <Routes language={language} />
    </main>
  );
};

export default Content;
