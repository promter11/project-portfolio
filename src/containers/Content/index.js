import { connect } from "react-redux";

import { getLanguage } from "../../store/selectors/LanguageSelectors";

import Content from "../../components/Content";

const mapStateToProps = (state) => {
  return {
    language: getLanguage(state),
  };
};

export default connect(mapStateToProps)(Content);
