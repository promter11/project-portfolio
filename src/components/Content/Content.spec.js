import React from "react";
import { shallow } from "enzyme";

import Content from "./index";

describe("Content component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Content language="RU" />);

    expect(wrapper).toMatchSnapshot();
  });
});
