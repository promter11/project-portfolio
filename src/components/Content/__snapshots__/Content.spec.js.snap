// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component renders correctly 1`] = `
<main
  className="main"
>
  <Routes
    language="RU"
  />
</main>
`;
